sources/documents/2_RTB/NormeDiProgetto/Supporto/documentazione.tex

```diff
@@ -139,11 +139,15 @@ \subsubsection{Stile}
 
 \paragraph{Formattazione testo}
 \begin{itemize}
-  \item \textbf{Termini nel Glossario:} Indicati in \textit{corsivo} e con una \textit{\small{G}} a fine parola. In base a ciascun documento tale formattazione può comparire alla sola prima occorrenza (quando il documento ha lo scopo di essere letto dall'inizio alla fine), o in maniera più frequente (quando il documento può essere letto in maniera più frammentata);
-  \item \textbf{Nomi di documento:} Indicati in \textit{corsivo} con le iniziali di parola maiuscole eccetto preposizioni (es.: \textit{Piano di Progetto}, non \textit{Piano Di Progetto}). Questo per mantenere la coerenza con le \glossario{abbreviazioni} (es: AdR, PdP, PdQ, NdP);
-  \item \textbf{Way of Working:} Indicato con le iniziali di parola maiuscole eccetto preposizioni, per mantenere la coerenza con l'\glossario{abbreviazione} WoW usata anche come comando in \glossario{LaTeX};
-  \item \textbf{Nomi di ruolo:} Indicati con la lettera iniziale maiuscola;
-  \item \textbf{Data:} Indicata in formato YYYY-MM-DD nelle tabelle riassuntive e nei nomi dei file, in formato esteso (esempio: 20 aprile 2024) quando si trova all'interno di testo discorsivo.
+  \item \textbf{Termini nel Glossario:} indicati in \textit{corsivo} e con una \textit{\small{G}} a fine parola. In base a ciascun documento tale formattazione può comparire alla sola prima occorrenza (quando il documento ha lo scopo di essere letto dall'inizio alla fine), o in maniera più frequente (quando il documento può essere letto in maniera più frammentata);
```

Comment:
```suggestion
  \item \textbf{Termini nel Glossario:} indicati in \textit{corsivo} e con una lettera \ped{G} in pedice a fine parola. In base a ciascun documento tale formattazione può comparire alla sola prima occorrenza (quando il documento ha lo scopo di essere letto dall'inizio alla fine), o in maniera più frequente (quando il documento può essere letto in maniera più frammentata);
```
Propongo di indicare che va in pedice (come nel modello di sezione del glossario già in uso)

Reply:
Ok